Content.Server/GameTicking/Presets/GamePreset.cs

```diff
@@ -65,7 +67,8 @@ public virtual bool OnGhostAttempt(Mind.Mind mind, bool canReturnGlobal)
                     if (playerEntity.TryGetComponent(out IDamageableComponent? damageable))
                     {
                         //todo: what if they dont breathe lol
-                        damageable.SetDamage(DamageType.Asphyxiation, 200, playerEntity);
+                        //cry deeply
+                        damageable.SetDamage(damageable.GetDamageType("Asphyxiation"), 200, playerEntity);
```

Comment:
Every day we descend one level deeper into potato.
This should be solved with a mob state component refactor probably.